doc/JSON_INFO.md

```diff
@@ -1344,6 +1344,46 @@ the item is spawned the variables set on the prototype no longer affect the item
 a migration can clear out the item's variables and reassign the prototype ones if reset_item_vars
 flag is set.
 
+### Item faults
+
+Faults can be defined for more specialized damage of an item.
+
+```C++
+{
+  "type": "fault",
+  "id": "fault_gun_chamber_spent", // unique id for the fault
+  "name": { "str": "Spent casing in chamber" }, // fault name for display
+  "description": "This gun currently...", // fault description
+  "item_prefix": "jammed", // optional string, items with this fault will be prefixed with this
+  "flags": [ "JAMMED_GUN" ] // optional flags, used by C++ parts
```

Comment:
Maybe add a link to the fault flags doc around here https://github.com/CleverRaven/Cataclysm-DDA/blob/a5245cacc5f4736bf272fce944dd9cf357483fe0/doc/JSON_FLAGS.md?plain=1#L1639

Reply:
Yep, linked and removed outdated parts from there as well